src/app/projects/services/projects.service.ts

```diff
@@ -129,7 +129,7 @@ export class ProjectsService implements OnDestroy {
     this.changePage(1);
   }
 
-  private getAllProjects(): Observable<Project[]> {
+  public getAllProjects(): Observable<Project[]> {
```

Comment:
This should be private. It is called by the constructor. Consumers of this service consume the public observables

Reply:
Ah wait, I see why you did this.

Reply:
This probably could be improved. I am sure when we revisit this we shall make it better.